packages/nc-gui/components/cell/SingleSelect.vue

```diff
@@ -120,9 +129,26 @@ async function addIfMissingAndSave() {
       })
       column.value.colOptions = { options: options.value.map(({ value: _, ...rest }) => rest) }
 
-      await $api.dbTableColumn.update((column.value as { fk_column_id?: string })?.fk_column_id || (column.value?.id as string), {
-        ...column.value,
-      })
+      const updatedColMeta = { ...column.value }
+
+      // todo: refactor and avoid repetition
+      // Postgres returns default value wrapped with single quotes & casted with type so we have to get value between single quotes to keep it unified for all databases
+      if (isPg.value) {
+        updatedColMeta.cdf = updatedColMeta.cdf.substring(
+          updatedColMeta.cdf.indexOf(`'`) + 1,
+          updatedColMeta.cdf.lastIndexOf(`'`),
+        )
+      }
```

Comment:
this would fail if you enter a tab with space.

![image](https://user-images.githubusercontent.com/35857179/203288299-85ada3ce-2117-49d7-9372-0649920bff5d.png)